source/_components/sensor.dnsip.markdown

```diff
@@ -31,14 +30,38 @@ sensor:
   - platform: dnsip
 ```
 
-Configuration variables:
-
-- **hostname** (*Optional*): The hostname for which to perform the DNS query. Defaults to `myip.opendns.com` (special hostname that resolves to your public IP).
-- **name** (*Optional*): Name of the sensor. Defaults to `myip` or hostname without dots if specified.
-- **resolver** (*Optional*): The DNS server to target the query at. Defaults to `208.67.222.222` (OpenDNS).
-- **ipv6** (*Optional*): Set this to `true` or `false` if IPv6 should be used. When resolving the public IP, this will be the IP of the machine where Home Assistant is running on.
-- **resolver_ipv6** (*Optional*): The IPv6 DNS server to target the query at. Defaults to `2620:0:ccc::2` (OpenDNS).
-- **scan_interval** (*Optional*): Defines number of seconds for polling interval. Defaults to `120` seconds.
+{% configuration %}
+hostname:
+  description: The hostname for which to perform the DNS query.
+  required: false
+  default: "`myip.opendns.com`" (special hostname that resolves to your public IP)
```

Comment:
The quote has to be close after the parentheses

Reply:
Thnx 😄